### PR TITLE
Generate special svg names.

### PR DESCRIPTION
**Pull request by keyurva**
* Mapping to schema names will be in a follow up PR.

### CODE REVIEW

File: simple/util/dc_client.py [hunk 1]
```diff
@@ -244,6 +244,50 @@ def resolve_entity_type(entity_dcids: list[str]) -> str:
...
+        _get_property_of_entities_chunk(client, chunk, property_name)
+        for chunk in chunks
+    ]
+    for result_chunk in await asyncio.gather(*futures):
+      result.update(result_chunk)
+
+  return result
+
+
+async def _get_property_of_entities_chunk(client: AsyncClient,
```
Comment (pradh) [inline simple/util/dc_client.py:269]: maybe this could be in a separate PR?

File: simple/stats/stat_var_hierarchy_generator.py [hunk 2]
```diff
@@ -136,7 +152,7 @@ def __init__(self, svg_id: str, svg_name: str) -> None:
     # Maintaining order maintains results consistency and helps with tests.
     self.sv_ids: dict[str, bool] = {}
     self.parent_svg_ids: dict[str, bool] = {}
-    self.child_svg_ids: dict[str, bool] = {}
+    self.child_svg_id_2_specialized_name: dict[str, str] = {}
```
Comment (pradh) [inline simple/stats/stat_var_hierarchy_generator.py:155]: How does this get surfaced?

**Review by pradh — APPROVED** [review]
nice!  this part has turned out to be a lot simpler here than in the flume!

File: simple/stats/stat_var_hierarchy_generator.py (hunk 2 above)
Comment (keyurva) [reply simple/stats/stat_var_hierarchy_generator.py:155]: Currently it does not. I'll generate a svg cache in a follow-on PR and it will get surfaced there.

File: simple/util/dc_client.py (hunk 1 above)
Comment (keyurva) [reply simple/util/dc_client.py:269]: huh - this _was_ in a separate [PR](https://github.com/datacommonsorg/import/pull/309). Let me see how it got here.

**Review by keyurva — COMMENTED** [review]
Thanks for the review!

File: simple/util/dc_client.py (hunk 1 above)
Comment (keyurva) [reply simple/util/dc_client.py]: OK - rebased correctly now so this is gone for this PR.
